SPECS/kmod-drbd.spec

```diff
@@ -1,7 +1,7 @@
-Name: drbd-kernel
+Name: kmod-drbd
 Summary: Kernel driver for DRBD
 Version: 9.2.10
-Release: 1
+Release: 1.0.xcpng.1
```

Comment:
You may use `%{?dist}` instead of a hardcoded `xcpng`. That would differentiate 8.2 and 8.3 package names.